Default.sublime-keymap

```diff
@@ -137,6 +128,15 @@
         ]
     },    // In case when auto match is enabled, only format if not within {}
     {
+        "keys": [ "alt+,"],
```

Comment:
I had to move this in order for the right keyboard shortcut to show in the quick panel.